matminer/featurizers/dos.py

```diff
@@ -6,6 +6,78 @@
 from pymatgen.electronic_structure.dos import CompleteDos, FermiDos
 
 
+class SiteDOS(BaseFeaturizer):
+    """
+    report the fractional s/p/d/f dos for a particular site. a CompleteDos
+    object is required because knowledge of the structure is needed. this
+    featurizer will work for metals as well as semiconductors. if the dos is a
+    semiconductor, cbm and vbm will correspond to the two respective band
+    edges. if the dos is a metal, then cbm and vbm correspond to above and
+    below the fermi level, respectively.
+
+    Args:
+        decay_length (float in eV):
```

Comment:
why not use a Fermi function for decay?

Reply:
Could be used, but after our discussion, I think the exponential decay is OK.

Reply:
ok